### DISCUSSION TITLE
Updated to preview 8 and having "Type 'MainWindow' already defines a member called 'InitializeComponent' with the same parameter types" errors

### DISCUSSION THREAD

**Discussion by JenyaRostov** [AvaloniaUI/Avalonia#11482] [Q&A] [answered]
![rider64_aMZTOavw4E](https://github.com/AvaloniaUI/Avalonia/assets/52300935/a4bf8735-14e0-45f6-bf57-1cfef7250631)

**Comment by timunie** [comment, marked as answer]
Read breaking changes wiki: https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes#xamlnamereferencegenerator-is-part-of-the-main-package-now

**Reply by JenyaRostov** [reply]
that fixed it, thanks a lot!
